### ISSUE TITLE
Keyboard navigation - esc to quit modal and highlighting CTAs

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/dataquality#145]
1. Can't use Esc key to get out of any modals (could be a design system level issue)
2. Check data button not highlighted in the same way as other buttons when tabbing through. This makes it hard to know that you have the button selected and that hitting enter will press the button

Highlighted state of the "Upload" tab:

![Image](https://github.com/user-attachments/assets/a5ddeb8e-6b6c-4a2b-a06a-ba41a9157576)

Highlighted state of the "Check data" button (not visible)

![Image](https://github.com/user-attachments/assets/679747d0-5985-4960-8c43-96863959d8f4)

Highlighted state of the "Submit your data for publication" link:

![Image](https://github.com/user-attachments/assets/97a0bd97-d747-4918-a008-36f5d5afee44)
